src/QmlControls/ParameterEditorController.h

```diff
@@ -141,9 +141,12 @@ private slots:
 private:
     bool _shouldShow(Fact *fact) const;
 
+    void _performSearch();
+
 private:
     ParameterManager*           _parameterMgr           = nullptr;
     QString                     _searchText;
+    QTimer*                     _searchTimer;
```

Comment:
Why does this need to be allocated?

Reply:
Simpler if just a regular member.

Reply:
Okay changed. I was just following the usage pattern I saw in other files.